.github/workflows/ci.yaml

```diff
@@ -14,21 +14,14 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu, macos, windows]
-        golang: ['1.13', '1.16', '1.17']
-        # currently, we cannot run non-x86_64 machines on Github Actions cloud env.
+        golang: ['1.19', '1.20']
```

Comment:
```suggestion
        golang: ['1.20', '1.21']
```

Reply:
updated to 1.19, 1.20, 1.21, as 1.21 is still quite new, so doesn't hurt to still keep 1.19 in the list.